improve CONTRIBUTING.md (#466)
OpenEnergyPlatform/ontology: CONTRIBUTING.md

### Contribution of OEO content
Please read the [OEO best practices](https://github.com/OpenEnergyPlatform/ontology/wiki/Best-Practice-Principles) carefully.

programs used

>  🔷 - git: used for syncronising the code on your pc and online

Reviewer: ```suggestion
 🔷 - git: used for synchronising the code on your PC and online
```

    ```bash
    git checkout -b feature/myfeature
    ```
    It is best to merge one's changes *as fast as possible* (i.e. do not wait for 2 months) to avoid merging conflicts

> 3. 📙 or 📗 Open [Protégé](https://protege.stanford.edu/) or a text editor and work on the ontology. If you haven't already, make sure you change your protégé settings to use [numeric identifiers](https://github.com/OpenEnergyPlatform/ontology/wiki/Numerical-Identifiers).

Reviewer: Somewhere here it has to be mentioned that the correct oeo module file has to be selected before adding new classes or relations.

Reviewer: Additionally one could mention here (maybe with the assistance of a screenshot) that for most cases one has to work in Protégé in the yellow tabs (_Entities_ and _Classes_). As we saw last week from the practical examples one major hurdle was to find in Protégé the correct place to work.

> 3. 📙 or 📗 Open [Protégé](https://protege.stanford.edu/) or a text editor and work on the ontology. If you haven't already, make sure you change your protégé settings to use [numeric identifiers](https://github.com/OpenEnergyPlatform/ontology/wiki/Numerical-Identifiers). Please choose the right module of the oeo to do your changes, oeo.omn is in most cases not the right file to change. Refer to [this article](https://github.com/OpenEnergyPlatform/ontology/wiki/how-to-use-prot%C3%A9g%C3%A9-to-change-the-ontology) for detailed explanations.

Reviewer: ```suggestion
3. 📙 or 📗 Open [Protégé](https://protege.stanford.edu/) or a text editor and work on the ontology. If you haven't already, make sure you change your protégé settings to use [numeric identifiers](https://github.com/OpenEnergyPlatform/ontology/wiki/Numerical-Identifiers). Please choose the right module of the oeo to do your changes, oeo.omn is in most cases not the right file to change. Refer to [this article](https://github.com/OpenEnergyPlatform/ontology/wiki/how-to-use-prot%C3%A9g%C3%A9-to-change-the-ontology) for detailed explanations on working with protégé.
```

    git checkout -b feature/myfeature
    ```

>     It is best to merge one's changes *as fast as possible* (i.e. do not wait for 6 months) to avoid merging conflicts

Reviewer 2: ```suggestion
    It is best to merge one's changes *as fast as possible* (i.e. do not wait for 2 months) to avoid merging conflicts
```

3. 📙 or 📗 Open [Protégé](https://protege.stanford.edu/) or a text editor and work on the ontology. If you haven't already, make sure you change your protégé settings to use [numeric identifiers](https://github.com/OpenEnergyPlatform/ontology/wiki/Numerical-Identifiers).

>     One can also edit the `oeo.omn` file in a text editor

Reviewer 2: This would be the right place to mention the modules I guess 
#464

4. 📙 Before committing your changes, open the `oeo.omn` file with Protégé and save the file from Protégé. You should also check if you included inconsistencies by following [this ontology test procedure](https://github.com/OpenEnergyPlatform/ontology/wiki/ontology-test-guide)

    See the "Conventions" section below for commit messages format tips
5. 🔷 Get your changes online
    
    stage the files you changed
    ```bash
    git add [file_name]
    ```

    commit your changes
    ```bash
    git commit [-m " <commit_message> "]
    ```

    push your branch to the remote server

    ```bash
    git push
    ```
    If your branch does not exist on the remote server yet, git will provide you with instructions, simply follow them

> 6. 🍏 Make sure that all automated tests are successful. This will be indicated by a green or red icon next to your most recent commit.

Reviewer 2: ```suggestion
6. 🍏 Make sure that all automated tests are successful. This will be indicated by a green icon next to your most recent commit. What to do in case an error occured, is described in section "Fixing merge conflicts".```

Author: The errors here are created by the changes itself, not merge conflicts. So I don't think this will help

Reviewer 2: Ok. But we should comment on what to do in such a case. Write a comment in the PR and ask for help from the ontology-expert-team?

Author: yes, good idea
